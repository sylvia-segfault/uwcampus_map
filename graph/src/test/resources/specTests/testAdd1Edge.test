# Create an empty graph
CreateGraph graph2

# Add a pair of nodes
AddNode graph2 node1
AddNode graph2 node2

# Add an edge to these two vertices in graph2
AddEdge graph2 node1 node2 edge1