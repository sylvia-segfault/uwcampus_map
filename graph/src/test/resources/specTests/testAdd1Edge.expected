# Create an empty graph
created graph graph2

# Add a pair of nodes
added node node1 to graph2
added node node2 to graph2

# Add an edge to these two vertices in graph2
added edge edge1 from node1 to node2 in graph2